storage/indexer_test.go

```diff
@@ -34,14 +36,57 @@ func BenchmarkInit(b *testing.B) {
 	defer fs.Stop()
 	storageClient := fs.Client()
 
-	logger := util.NewTestLogger()
+	logger := util.NewTestLoggerLevel(zapcore.FatalLevel)
+	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
 		indexer := NewIndexer(logger, storageClient, FakeIndexerOptions)
 		err := indexer.Init(context.Background())
 		require.NoError(b, err)
 	}
 }
 
+func BenchmarkIndexerUpdateIndex(b *testing.B) {
```

Comment:
Thinking about this (and the other benchmark added in this PR), not sure if this would give us the necessary information.

IIUC these benchmark results would give us the total amount of memory used without considering the memory that has been freed.

WDYT?